tests/integration/test_kubectl_delete.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
```

Comment:
In the PR message, can you please explain the reason behind this change?

Reply:
actually as part of my OB i have to add test coverage in pytest i asked Chris and he said ok i can add kubcectl delete as test coverage in this code as part of my Onboarding at canonical

Reply:
You mean in License i have no idea honestly i copied the file from other charm and leave the license part as it is

Reply:
This test is an integration test so it won't make a difference in code coverage and I don't want to add a test just for the sake of it, we'll have to maintain them once they're added. That being said, if we find places where there's a behaviour that isn't validated in the current tests, yes let's add tests to validate it.

Reply:
actually i was missing the test to delete the pods like in other integration tests . can we figure out any solution to add above ?